Apps/BMSQueryApp/app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
this file can be removed?

Reply:
This file cannot be removed. It is used to shrink down the size of the apk. Although there is nothing in this file, after removing it I will not be able to run the app with the IDE, so it is better to keep it.

Reply:
Okay, noted.